roles/engineblock/templates/parameters.yml.j2

```diff
@@ -1,4 +1,115 @@
 parameters:
+    domain: {{ base_domain }}
+    hostname: engine.{{ engine_base_domain }}
+    trusted_proxies:
+{% for engine_trusted_proxy_ip in engine_trusted_proxy_ips %}
+        - {{ engine_trusted_proxy_ip }}
+{% endfor %}
+    encryption_keys:
+{% for key, value in engine_keys.items() %}
+        {{ key }}:
+            publicFile: {{ value.publicFile }}
+            privateFile: {{ value.privateFile }}
+{% endfor %}
+    forbidden_signature_methods: {  }
+    allowed_acs_location_schemes:
+        - http
+        - https
+    php_settings:
+        memory_limit: 128M
+        display_errors: '{{ php_display_errors }}'
+        error_reporting: '6135'
+        date.timezone: {{ timezone }}
+        sendmail_from: 'OpenConext EngineBlock <openconext-engineblock@openconext.org>'
```

Comment:
Really should be `{{ instance_name }} <{{ support_email }}>`

Reply:
Be aware that this seems to change the original behaviour because it didn't seem to be defined in the `engineblock.ini.j2` in deploy and was coming from the default `application.ini` in EB itself. Or does it come from somewhere else and do I missing something here myself?

Reply:
No just observing that we should be avoiding this kind of hardcoded values. Can indeed be that it was already hardcoded earlier.

Reply:
I've changed it as suggested.